packages/patternfly-3/react-console/less/desktop-viewer.less

```diff
@@ -0,0 +1,52 @@
+.desktop-viewer-pf {
+  > .row {
+    display: flex;
+    flex-wrap: wrap;
+  }
```

Comment:
Changing row to be display: flex and using col-md-6 on the children is producing unexpected results.
Opening and closing the `More Information` is jumping the `manual connection-pf` div around.

Reply:
Thanks, good catch. Fixed.